test/gtest/uct/uct_test.cc

```diff
@@ -494,15 +525,30 @@ bool uct_test::get_config(const std::string& name, std::string& value) const
     const size_t max = 1024;
 
     value.resize(max);
-    status = uct_config_get(m_iface_config, name.c_str(),
-                            const_cast<char *>(value.c_str()), max);
 
-    if (status == UCS_ERR_NO_ELEM) {
-        status = uct_config_get(m_md_config, name.c_str(),
+    if (m_cm_config != NULL) {
+        status = uct_config_get(m_cm_config, name.c_str(),
+                                const_cast<char *>(value.c_str()), max);
+        if (status == UCS_OK) {
+            return true;
+        }
+    }
+
+    if (m_iface_config != NULL) {
+        status = uct_config_get(m_iface_config, name.c_str(),
                                 const_cast<char *>(value.c_str()), max);
+        if (status == UCS_OK) {
+            return true;
+        }
     }
 
-    return (status == UCS_OK);
+    status = uct_config_get(m_md_config, name.c_str(),
```

Comment:
no need to check for NULL?
AFAIR we don't have MD in case of CM?

Reply:
@yosefe m_md_config is always present - https://github.com/openucx/ucx/blob/master/test/gtest/uct/uct_test.cc#L148
currently rdmacm is on the uct_rdmacm_component which has an md.

Reply:
ok, looks like we won't even instantiate tests on a component without MD resources, so can assume MD exists for now